### ISSUE TITLE
Issues with themablebrowser

### ISSUE THREAD

**Issue by vespino** [EddyVerbruggen/SocialSharing-PhoneGap-Plugin#392]
Hi Eddy,

Are you familiar with this issue? https://github.com/initialxy/cordova-plugin-themeablebrowser/issues/44

**Comment by EddyVerbruggen** [comment]
Hi, didn't see this before, but indeed this should be fixable after all. We just need to find the currently displayed viewcontroller and have that present the share sheet.

This should work, I'm testing it right away:
```
UIViewController *presentingViewController = [[[UIApplication sharedApplication] delegate] window].rootViewController;
  while (presentingViewController.presentedViewController != nil) {
    presentingViewController = presentingViewController.presentedViewController;
  }
[presentingViewController presentViewController:activityVC animated:YES completion:nil];
```

**Comment by EddyVerbruggen** [comment]
Fixed in 4.3.20, thanks for getting my attention!
